### PR TITLE
Adding a working container for NanoSV

### PR DESCRIPTION
**Pull request by oneillkza**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [ ] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [ ] The image can be built  
 - [ x] The BioConda package is missing a dependency (sambamba), which this container provides.
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [X ] LABEL base_image  
 - [X ] LABEL version
 - [X ] LABEL software.version  
 - [X ] LABEL about.summary  
 - [X ] LABEL about.home  
 - [X ] LABEL about.license   
 - [X ] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata  

As noted above, while this is available as an autobuild from BioConda, the tool itself actually depends on sambamba, but that dependency is not listed in the BioConda package. The result is that the autobuilt container is not self-contained. This container adds sambamba, so that it is now self-contained. The python version had to be specified due to some dependency issues (see https://github.com/mroosmalen/nanosv/issues/65).

### CODE REVIEW

**Comment by osallou** [comment]
can't modify multiple containers in a same pull request

**Comment by osallou** [comment]
can't modify multiple containers in a same pull request

**Comment by osallou** [comment]
can't modify multiple containers in a same pull request

**Comment by oneillkza** [comment]
oops -- accidentally committed a vim swapfile. Should be good now.

**Comment by oneillkza** [comment]
PS: this software is basically abandonware. I am not the software maintainer. I just want Biocontainers to contain a version of it that's actually, well, contained.

File: nanosv/1.2.4/Dockerfile
Comment (osallou) [inline nanosv/1.2.4/Dockerfile]: This biocontainers doc page ;-) should siftware doc page, if any, else put homepage

**Review by osallou — CHANGES_REQUESTED** [review]

**Comment by osallou** [comment]
Thanks for contrib. There is just a small update to be done before merge

**Comment by oneillkza** [comment]
No worries! And I've fixed that. To be honest the sum-total of the documentation is the readme.md on the Github repo.

Hopefully it should be good to go!

**Review by osallou — APPROVED** [review]

**Pull request merged**
